### ISSUE TITLE
Triggering with a keyboard shortcut

### ISSUE THREAD

**Issue by infokiller** [chromium/dom-distiller#12]
Is it possible to trigger with a keyboard shortcut?
If not, what do you think would be the best approach to implement it? is an extension a reasonable way to do it?

Thanks!

**Comment by infokiller** [comment]
@wychen can you please chime in?
(if you're not the right address for this, please let me know and I'll forward it to the right person)

**Comment by wychen** [comment]
DOM distiller is the content extraction backend. This is used in Chrome on Android, for the feature "simplified view". There are 3rd party Chrome Extensions based on this project. Are you using a developer extension described in the doc here? https://github.com/chromium/dom-distiller#developer-extension

**Comment by infokiller** [comment]
No, I'm not using that extension. I'm actually not working with the code at all- I use the DOM distiller from the stable version of Chromium.
I'm looking for a way to trigger the "Distill page" by using a keyboard shortcut, without having to go to the Chromium menu.

**Comment by infokiller** [comment]
@wychen you previously mentioned 3rd part extensions based on this project.
Can you please reference them?

I'd like to be able to trigger dom distiller with a keybinding, so if I can write an extension to use it I'll be able to do it.

Thanks
